### PR TITLE
Gen cleanup

### LINKED ISSUE #81: Generated code is pretty ugly, should clean it up
And maybe name the file `{{symbol}}.sol` instead of name.sol. 

Screenshot 2019-03-14 at 9 40 02 AM

### PR DESCRIPTION
**Pull request by treeder**
Cleans up the generated code.

Screenshot 2019-03-19 at 10 43 48 AM

Closes #81

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -2,8 +2,9 @@ UNAME_S := $(shell uname -s)
 build: 
 	go build ./cmd/web3
 
-install:
-	go install ./cmd/web3
+install: build
+	# go build -o /usr/local/bin/web3 ./cmd/web3
+	sudo cp web3 /usr/local/bin/web3
```
Comment (jmank88) [inline Makefile]: `sudo` seems heavy handed

Comment (jmank88) [reply Makefile]: And it's commented out? Leftover hack?

Comment (treeder) [reply Makefile]: sudo is the only way to install it over the main installer. 

Removed comment.

Comment (jmank88) [reply Makefile]: I'm not following. Why wasn't `go install` working?

Comment (treeder) [reply Makefile]: Well just tried again, thinking the installed web3 was overriding the one that `go install` makes and throws into ~/go/bin. But looks like I also don't have ~/go/bin in my path so there's that. 😆  But the way it's done here ensures that you won't have conflicting versions installed if you installed it via the install.sh script too.

**Pull request merged**